src/emoji/data.js

```diff
@@ -20,6 +21,19 @@ export const codeToEmojiMap: {| [string]: string |} = objectFromEntries<string,
   }),
 );
 
+// TODO(?): Stop having distinct `EmojiType` and `ReactionType`; confusing?
+//   https://github.com/zulip/zulip-mobile/pull/5269/commits/dda8972eac9175b4fc2d3a16255280768025e98a#r818320669
```

Comment:
Remember that these links with both a PR number and a commit ID are unstable; this one is now broken.

I think the link you want is this one:
https://github.com/zulip/zulip-mobile/pull/5269#discussion_r818320669
(Which I guess suggests that there's a systematic way to transform a broken link of this form into the stable version, so that's good at least.)

Reply:
Eep, yep, thanks for the reminder!

> Which I guess suggests that there's a systematic way to transform a broken link of this form into the stable version, so that's good at least.

Cool, yeah, I see. I'll remember that transformation and pay better attention in future. 🙂